doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/project-storage-version-build-options-ref.adoc

```diff
@@ -0,0 +1,160 @@
+[id='project-storage-version-build-options-ref_{context}']
+= Project storage and build options with {PRODUCT}
+
+As you develop a {PRODUCT} project, you need to be able to track the versions of your project with a version-controlled repository, manage your project assets in a stable environment, and build your project for testing and deployment. You can use {CENTRAL} for all of these tasks, or use a combination of {CENTRAL} and external tools and repositories. {PRODUCT} supports Git repositories for project version control, Apache Maven for project management, and a variety of Maven-based, Java-based, or custom-tool-based build options.
+
+The following options are the main methods for {PRODUCT} project versioning, storage, and building:
```

Comment:
I think that each table should have a more elaborated content. I could digest the first two tables, but the thirs (Project build options) I felt overwhelmed.

Reply:
"more elaborated". Do you mean more condensed? The problem I face here is that I have to give enough info to be helpful to the user, and then for the rest, direct to the full documentation. This means that for each option, I need to give the option, describe it, and point to the docs related to it in a clear way that can be skimmed quickly. I played around with various ways of doing this (paragraphs, bullets, sections) and those were all more overwhelming or cluttered-feeling, no matter how streamlined I tried to make it. Particularly because several options have multiple doc links, which looks rough in a non-tabular format (I tried it). This tabular format is the least dense-looking and better than the alternatives for quickly skimming for the various options and what they mean, as well as the multiple doc links that need to accompany some of the entires. If you have a specific alternative you'd like to see, I can show you, but honestly, this was the cleanest and clearest, imo.